Draft-OpenChain-Telco-SBOM-Guide_EN.md

```diff
@@ -119,10 +119,10 @@ Most SCA tools have the capability to produce hashes.
 Package URL (PURL) is a _de facto_ standard to uniquely identify software packages.
 
 ### 3.3 Machine Readable Data Format
-The Telco SBOM SHALL include, at a minimum, the SPDX in the following machine readable format as default: Tag:Value
+The Telco SBOM SHALL include, at a minimum, the SPDX in one of the following machine readable formats: Tag:Value or JSON.
 
 #### 3.3.1 Verification and reference material
-Tag:Value is described here in SPDX 2.2 https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements
+Tag:Value and JSON formats are described here in SPDX 2.2 https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements
```

Comment:
I did not see any difference between SPDX 2.2 and 2.3 for **4.4 Standard data format requirements**, but if SPDX 2.3 will be added, then should we also include a link to SPDX 2.3 spec?
Link: https://spdx.github.io/spdx-spec/v2.3/conformance/#44-standard-data-format-requirements

Reply:
Since the definition in 2.2 and 2.3 is the same, I had only included one on purpose.

I have now added the link to 2.3.

Reply:
Yep, I did not see any difference between the two, but someone not knowing that there are not differences, may be confused.